packages/app/src/files-manager/components/files-manager/files-manager.interface.tsx

```diff
@@ -6,23 +6,16 @@
 import { Type } from "Renderer/components/core/icon/icon.config"
 import { FilesType } from "App/files-manager/constants/files-manager.enum"
 
-export interface DiskSpaceCategory extends Item {
+export interface DiskSpaceCategory {
   filesType: FilesType
-  occupiedMemory?: number
+  megabyteSize?: number
   filesAmount?: number
   color: string
   icon: Type
+  free?: boolean
 }
 
 export interface MemorySpaceCategory {
```

Comment:
Why here is a `Category` suffix?

Reply:
Removed in [ce40a64](https://github.com/mudita/mudita-center/pull/598/commits/ce40a6429870732cb157448924ac3de654ac19c3)